week2/basicsorting.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script>
+    let arraytosort = [];
```

Comment:
Since we are not reassigning the array to another value, we can actually use `const` here instead.